modules/graphql_core/src/Annotation/GraphQLType.php

```diff
@@ -19,7 +19,7 @@ class GraphQLType extends GraphQLAnnotationBase {
    *
    * @var string
    */
-  public $data_type;
+  public $dataType;
```

Comment:
Why did you make this camel case? It's a plugin definition property. I thought we agreed to keep them in snake case?